sdk/communication/AzureCommunicationCommon/Source/Identifiers.swift

```diff
@@ -25,16 +25,91 @@
 // --------------------------------------------------------------------------
 
 import Foundation
+/**
+ The IdentifierKind for a given CommunicationIdentifier.
+ */
+
+@objcMembers public class IdentifierKind: NSObject {
+    private var kindValue: String
+    public static let communicationUser = IdentifierKind("communicationUser")
+    public static let phoneNumber = IdentifierKind("phoneNumber")
+    public static let microsoftTeamsUser = IdentifierKind("microsoftTeamsUser")
+    public static let unknown = IdentifierKind("unknown")
+
+    public init(_ kindValue: String) {
```

Comment:
This aligns with other raw representable enums.
```suggestion
    public init(rawValue: String) {
```

Reply:
Changed from `kindValue` to `rawValue`